libecs/include/pivot/ecs/Core/Component/description.hxx

```diff
@@ -68,3 +68,9 @@ struct Description {
     bool operator==(const Description &rhs) const;
 };
 }    // namespace pivot::ecs::component
+
+namespace pivot::builtins::components
+{
+template <typename T>
+extern const ecs::component::Description description;
+}
```

Comment:
imo,  you should avoid `extern` declaration. It makes the code clearer.

Reply:
This is dead code. I removed it